ai_genomics/analysis/data_scoping/patent_data/search_evaluation.md

```diff
@@ -0,0 +1,64 @@
+
+### Evaluation
+
+One hundred examples were randomly sampled from the data and inspected to determine whether they related to AI or genomics (`outputs/data/patents/ai_genomics_patents_verified.csv`). Their title and abstract were read to see if they contained content relating to those subjects and binary indicators were used to denote whether or not this was the case. In some cases, the title and abstract were ambiguous, and the patent was searched for on Google Patents and inspected in further detail. This was typically done by skim reading and searching for keywords belonging to each topic (e.g. gene, genetic, dna, nucleic, machine learning, classify, cluster, detect). The results should be interpreted with some degree of error as it is often quite difficult to identify whether a patent relates to the topics of interest due to vague language, non-expert knowledge in genomics and the length of some patents.
+
+The table below shows the counts of projects in the sample that are and are not related to AI and genomics. It is immediately clear that there are many projects that are not related to the two subjects - 66 are related to only one of them, and 7 are related to neither. The content of the titles, abstracts and patent texts suggests some reasons as to why this may be.
+
+|        | Not Genomics | Genomics |
+| :----- | -----------: | -------: |
+| Not AI |            7 |       41 |
+| AI     |           25 |       27 |
+
+One reason is that the definitions of AI and genomics are overly broad. The current filtering mechanism includes patents predicted to relate to machine learning, natural language processing, computer vision, speech, knowledge processing, AI hardware, evolutionary computation, and planning and control. Several patents covered topics such as new silicon chip architectures and automated manufacturing processes, which are more tangentially related to enabling and employing AI. In addition, there were several patents that covered genetic algorithms, but did not touch upon genomics. Upon closer inspection, the patent codes for these documents often referred to 'genetic computing'.
+
+Another possible reason is poor classification of AI topics in the USPTO data. Several examples in the dataset discussed topics that contained vocabulary associated with AI and machine learning but that were being used in other contexts. For example, one patent referred to 'clustering' several times but was in fact referring to the physial aggregation of biological particles. In several cases, the word 'data' is used throughout, but the methods described to use the data are not based on machine learning. There were also some patents that used conventional computer vision, reliant on conventional image processing rather than AI-like technologies. It is not immediately clear which part of the pipeline used to predict AI patents that these errors originate from.
```

Comment:
yes, we came across this issue as well i.e. neural networks used a lot throughout a patent but was referring to its original definition of biological neurons